### ISSUE TITLE
Support for Hive/Impala/SparkSQL based Tables

### ISSUE THREAD

**Issue by STHITAPRAJNAS** [tokern/data-lineage#10]
It would be nice to have support of more databases. May be a generic JDBC way or including pluggable 3rd party drivers.

**Comment by vrajat** [comment]
Definitely. Right now every database has a different method to get query history. For example, Hive or SparkSQL itself does not show query history. 

* Which distribution do you use? 
* Do you run sparksql on the cloud?
* Do you store queries that are run in sparksql ?

I am wondering if I should also build a module to capture queries from these data engines.
